go.sh

```diff
@@ -2,15 +2,22 @@
 
 set -e
 
+# This follows a different path to the other tests/shell, since the e2e script
+# spins up its own containers as needed.
+if [ $1 = "e2e" ]
+then
+  ./bin/e2e.sh
+  exit
+fi
+
 # Usage examples:
 # `./go.sh` to get an interactive shell in a container for development
 # `./go.sh yarn test` to run the tests in a container and then exit
-
 docker-compose up -d --remove-orphans
 
 if [[ -z $1 ]];
 then
-  docker-compose exec core /bin/sh
+  docker-compose exec core /bin/bash
```

Comment:
Did you test this change? I think you'll find that bash is not present in Alpine linux, which is what the `core` image is built from. So it needs to be `sh`.

Reply:
I checked this change against the alpine image and it seemed fine - I suppose I can revert it.